src/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -1069,7 +1069,7 @@ class as a vector. Label ordering is dictated by the ``classes``
             Rows: 10
             [3, 4, 3, 3, 4, 5, 8, 8, 8, 4]
         """
-        feature_column = self.__proxy__.get_state()["feature_column_name"]
+        feature_column = self.features[0]
```

Comment:
Per offline discussion, let's preserve the structure of the model state from 5.8